examples/shors.rs

```diff
@@ -44,7 +48,7 @@ fn main() {
             d2.to_string().yellow()
         );
         let t = sw.elapsed_ms();
-        runtimes.push(t);
+        if t > 1 {runtimes.push(t);} 
```

Comment:
This causes divide by 0 when not using circuit

Reply:
Well it turns out I was wrong: It doesn't cause divide by 0 when not using circuit, it causes divide by 0 when the correct factors are guessed.

To clarify: `d1` is not zero, the problem happens on the following line
```
    let avg = runtimes.iter().sum::<i64>() / runtimes.len() as i64;
```
because if the factors are guessed the program takes 0 ms, and so because of the highlighted code `runtimes.len() == 0`

Reply:
Oh i see